src/bin/taker.rs

```diff
@@ -117,11 +128,20 @@ fn main() {
     )
     .unwrap();
 
+    let log_level = match args.command {
+        Commands::DoCoinswap | Commands::SyncOfferBook | Commands::SendToAddress { .. } => {
+            log::LevelFilter::Info
+        }
+        _ => log::LevelFilter::Off,
+    };
```

Comment:
TODO: Log should be toggled with a `--verbosity` argument. Can be done in a followup PR.